sql.txt

```diff
@@ -1,14 +1,26 @@
 (1) Using the Customer Table, select the CustomerName, ContactName, and Country.  Use ORDER BY to order by Country.  
 Use LIMIT and OFFSET to get entries 11 through 20. Paste your SQL statement below. 
 
+    SELECT CustomerName, ContactName, Country FROM Customers ORDER BY Country LIMIT 10 OFFSET 10;
+
 (2) Select all columns from the Customer table where the ContactName starts with A.  Paste your SQL statement below.
 
+    SELECT * FROM Customers ORDER BY ContactName ASC;
+
 (3) Select all columns from the OrderDetails table where the ProductID is 51 and the quantity is greater than 10.
 Paste your SQL statement below.
 
+    SELECT * FROM OrderDetails WHERE ProductID = 51 AND Quantity > 10;
+
 (4) Insert 3 rows into the Products table.  Note that you will have to specify a valid SupplierID and CategoryID, 
 corresponding to rows from the Supplier and Category tables.  Paste your three SQL statements below.
 
+    INSERT INTO Products(ProductName, SupplierID, CategoryID) VALUES ("Amy", 2, 1);
```

Comment:
So you inserted 1 row into the table, but the problem asks for 3 total. Try adding 2 more rows into the table.

Reply:
`insert into Products (ProductName, SupplierID, CategoryID) values ("Amy", 1, 2),("Tom", 2, 3), ("Joe", 4, 5);`